NOTICE

```diff
@@ -14,7 +14,8 @@ Copyright 2008- Deutsches Zentrum für Luft- und Raumfahrt e.V. (DLR) and partne
 ==========================================
 (1) ESMValTool VERSION 2.0 CORE DEVELOPMENT TEAM
 ==========================================
-Copyright 2008- Deutsches Zentrum für Luft- und Raumfahrt e.V. (DLR), Germany - ESMValTool Principal Investigator (PI)
+Copyright 2008- Deutsches Zentrum für Luft- und Raumfahrt e.V. (DLR), Germany - ESMValTool Co-PI
+Copyright 2020- Met Office, UK - ESMValTool Co-PI
 Copyright 2017- Alfred-Wegener-Institute (AWI), Germany
 Copyright 2017- Barcelona Supercomputing Center (BSC), Spain
 Copyright 2016- Ludwig Maximilian University, Germany
```

Comment:
```suggestion
Copyright 2016- Ludwig Maximilian University, Germany
Copyright 2011- Swedish Meteorological and Hydrological Institute (SMHI), Sweden
```

Reply:
Sorry, this line should be added a bit lower, below `Copyright 2019- Plymouth...`, but I couldn't comment there in the github GUI.

Reply:
Done!